startleft/cli.py

```diff
@@ -135,5 +161,18 @@ def search(type, query, filename):
     logger.info("Running JMESPath search query against source files")
     iac_to_otm.search(type, query, filename)
 
+
+@click.option('--server', default='http://localhost:8080', envvar='IRIUS_SERVER',
+              help='IriusRisk server to connect to (proto://server[:port])')
+@cli.command()
+def server(server: str):
+    """
+    Launches the REST server in development mode to test the API
+    """
+    from startleft.api import fastapi_server
+    webapp = fastapi_server.initialize_webapp(server)
+    uvicorn.run(webapp, host="127.0.0.1", port=5000, log_level="info")
```

Comment:
Externalize these parameters, at least the port, and maybe the IP if you want to only listen for IPv6?